tests/c/phi3.newcg.c

```diff
@@ -1,40 +1,7 @@
 // ignore-if: test $YK_JIT_COMPILER != "yk" -o "$YKB_TRACER" = "swt"
 // Run-time:
-//   env-var: YKD_LOG_IR=-:aot,jit-pre-opt
 //   env-var: YKD_LOG_JITSTATE=-
 //   env-var: YKD_LOG_STATS=/dev/null
-//   stderr:
```

Comment:
This seems a bit of a drastic change. This essentially removes everything we're trying to test. What was failing about this test?

Reply:
oh this is an accidental push, will correct this.